### PR TITLE
Feature - change folder login

### PR DESCRIPTION
**Pull request by at-dungduong**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: Demo/Login/Login/Tabbar/HomeViewController.swift [hunk 1]
```diff
@@ -123,6 +113,20 @@ class HomeViewController: UIViewController, MVVM.View {
         categoriesCollectionView.reloadData()
         viewDidUpdated()
     }
+
+    // MARK: - @objc
+    @objc func scrollToNextCell() {
+        let cellSize = CGSize(width: promotionsCollectionView.frame.width, height: promotionsCollectionView.frame.height)
```
Comment (at-tuanmai2) [inline Demo/Login/Login/Tabbar/HomeViewController.swift]: please dèine side

File: Demo/Login/Login/Tabbar/HomeViewController.swift [hunk 2]
```diff
@@ -123,6 +113,20 @@ class HomeViewController: UIViewController, MVVM.View {
...
+        let contentOffset = promotionsCollectionView.contentOffset
+        if promotionsCollectionView.contentSize.width <= contentOffset.x + cellSize.width {
```
Comment (at-tuanmai2) [inline Demo/Login/Login/Tabbar/HomeViewController.swift]: in viewModel

File: Demo/Login/Login/Tabbar/HomeViewController.swift [hunk 3]
```diff
@@ -151,7 +155,7 @@ extension HomeViewController: UICollectionViewDelegate {
         if collectionView == promotionsCollectionView {
             let detailVC = DetailViewController()
             guard let id = viewModel.promotion?[indexPath.row].id else { return }
-            detailVC.viewModel.id = id
+            detailVC.viewModel = viewModel.detailViewModel(id: id)
```
Comment (at-tuanmai2) [inline Demo/Login/Login/Tabbar/HomeViewController.swift]: viewmodelWitch

File: Demo/Login/Login/Tabbar/MapViewController.swift [hunk 4]
```diff
@@ -11,26 +11,51 @@ import MapKit
...
         mapView.delegate = self
         AppDelegate.shared.configLocationService()
         mapView.showsUserLocation = true
     }
-    
+
+    // MARK: - Private
+    private func configNaviBar() {
+        navigationController?.navigationBar.isHidden = false
+        navigationController?.navigationBar.barTintColor = #colorLiteral(red: 0.8378158212, green: 0.1756822765, blue: 0.232298106, alpha: 1)
```
Comment (at-tuanmai2) [inline Demo/Login/Login/Tabbar/MapViewController.swift]: define color

**Pull request merged**